packages/playwright-crawler/src/internals/utils/playwright-utils.ts

```diff
@@ -811,7 +827,17 @@ export function registerUtilsToContext(
         await injectJQuery(context.page, { surviveNavigations: false });
     };
     context.blockRequests = async (options?: BlockRequestsOptions) => blockRequests(context.page, options);
-    context.parseWithCheerio = async () => parseWithCheerio(context.page, crawlerOptions.ignoreShadowRoots);
+    context.waitForSelector = async (selector: string, timeoutMs?: number) => {
+        const locator = context.page.locator(selector).first();
+        await locator.waitFor({ timeout: timeoutMs, state: 'attached' });
```

Comment:
This does not throw, as opposed to the HTTP/cheerio counterpart. Is this intentional?